README.md

```diff
@@ -1 +1,18 @@
 # study-app
+
+This app is using commons-ui library which released in the github packages.
+
+To run the app you should first connect to github packages registry
+
+  > npm login --registry=https://npm.pkg.github.com
```

Comment:
But we don't need to login to just get commons-ui dependency?

Reply:
we need to login the github packages firs!